planetarium/NineChronicles
# Improve to equip costumes in the quest preparation UI

### Description

1. changes : `QuestPreparation.cs`, `UI_QuestPreparation.prefab`
2. In the past, player can not equip costumes in quest preparation UI.
3. Now, player can equip costumes in quest preparation UI.

### How to test

1. Enter in quest preparation UI.
2. Check can equip/un-equip costumes.

### Related Links

[코스튬 착용을 전투준비에서도 할 수 있도록](https://app.asana.com/0/958521740385861/1199239346057445/f)

### Screenshot
![210809 equip costumes in quest preparation](https://user-images.githubusercontent.com/48484989/128674170-d0f81dd6-db06-4c90-bb0e-77b3b6e06e85.gif)
